libraries/abstractions/common_io/test/test_iot_watchdog.c

```diff
@@ -49,10 +49,18 @@
 #define testIotWATCHDOG_DEFAULT_SEMAPHORE_DELAY    100000U
 /*-----------------------------------------------------------*/
 
+/*-----------------------------------------------------------*/
+
+/* Globals values which can be overwritten by the test
+ * framework invoking these tests */
+/*-----------------------------------------------------------*/
+uint8_t ucTestIotWatchdogInstance = 0; /** Default Flash instance for testing */
+
 /*-----------------------------------------------------------*/
 /** Static globals */
 /*-----------------------------------------------------------*/
-static SemaphoreHandle_t xtestIotWatchdogSemaphore = NULL;
+static SemaphoreHandle_t xtestIotWatchdogBarkSemaphore = NULL;
+static SemaphoreHandle_t xtestIotWatchdogBiteSemaphore = NULL;
```

Comment:
What is bark and bite?

Reply:
As per the API description, Bark is the alert to "feed" the watchdog. Bite is the watchdog actually expiring.